src/scripts/scaling/scalingAction.sh

```diff
@@ -2,24 +2,272 @@
 # Copyright (c) 2017, 2020, Oracle Corporation and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-echo "called scalingAction.sh" >> scalingAction.log
+currdate=`date`
```

Comment:
1) Nice to have: See resources/scripts/utils.sh for a timestamp bash function that generates a timestamp in a format that exactly matches the operator logs timestamp format and the timestamp format we use in startServer.sh, etc.  (It's nice to have all of our major scripts use the same timestamp format.)
2) Nice to have: Consider defining a 'trace' function to replace your use of echo throughout. The trace can then automatically inject the timestamp...

Reply:
Would you be ok if I just pull in the trace and timestamp functions since this is a standalone script, that's not deployed or run as part of the operator, where user's have to copy to use it either in a webhook or with WLDF to use  for scaling?

Reply:
> Would you be ok if I just pull in the trace and timestamp functions since this is a standalone script, that's not deployed or run as part of the operator, where user's have to copy to use it either in a webhook or with WLDF to use for scaling?

Sure. That's definitely better IMO.

Feel free to copy the timestamp function. 

I have some doubt that you want to copy over the full trace function - something like the following would be fine IMO: 
```
function traceInfo() {
   echo "@[$(timestamp)][INFO]" "$@"
}
function traceError() {
   echo "@[$(timestamp)][ERROR]" "$@"
}
```

Reply:
LGTM